### DISCUSSION TITLE
Error Page in build folder

### DISCUSSION THREAD

**Discussion by TJMusiitwa** [schultek/jaspr#235] [Q&A] [answered]
Hi @schultek 
Excited to be able to use the package and have been able to build up a website relatively quickly. The only slight issue and one that I am not sure how to solve is to locate the error page that I have defined when building and deploying the website. 

Checking the build folder and it seems that the error page is not among those that is generated.

I have my App router defined as below;
```
Router(
        routes: [
          ShellRoute(
            builder: (context, state, child) =>
                Builder(builder: (context) sync* {
              yield div([
                Header(),
                child,
                Footer(),
              ], classes: 'relative');
            }),
            routes: [
              Route(
                  path: '/',
                  title: 'Home',
                  builder: (context, state) => const Home()),
              Route(
                  path: '/about',
                  title: 'About',
                  builder: (context, state) => const About()),
              Route(
                  path: '/contact-us',
                  title: 'Contact Us',
                  builder: (context, state) => ContactUs()),
            ],
          ),
        ],
        errorBuilder: (context, state) => Error404(),
      ),
```

Running the website using `jaspr serve` on localhost I am able to see the error page if I enter a route that is not defined. 

The reason I ask is after deploying the website to Azure Static WebSite Hosting, they mention being able to [override the page](https://learn.microsoft.com/en-us/azure/static-web-apps/configuration#response-overrides) shown on an error.

It would be great to be able to use my nicely designed error page and not the default one that Azure provides. 

Many Thanks.

**Comment by schultek** [comment, marked as answer]
I would suggest adding an additional route with a '/404` path. Then you can point azure to that generated '/404/index.html' file.

**Reply by TJMusiitwa** [reply]
Yup that is what I sort of went with. Thanks
